pom.xml

```diff
@@ -116,5 +116,9 @@
             <artifactId>credentials-binding</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>ssh-credentials</artifactId>
+        </dependency>
```

Comment:
This should not be needed.
```suggestion
```

Reply:
I agree with @MarkEWaite  you should not need additional dependency.